data/infra/resources/directory.yaml

```diff
@@ -156,79 +162,7 @@ properties_list:
   required: false
   default_value: 'false'
   description_list:
-  - markdown: Create or delete parent directories recursively. For the owner, group,
-      and mode properties, the value of this property applies only to the leaf directory.
-- property: rights
-  ruby_type: Integer, String
-  required: false
-  description_list:
-  - markdown: 'Microsoft Windows only. The permissions for users and groups in a
-
-      Microsoft Windows environment. For example:
-
-      `rights <permissions>, <principal>, <options>` where `<permissions>`
-
-      specifies the rights granted to the principal, `<principal>` is the
-
-      group or user name, and `<options>` is a Hash with one (or more)
-
-      advanced rights options.'
```

Comment:
Should this be deleted?

Reply:
Yeah something is weird here. This shouln't be deleted.

Reply:
Currently published docs are also missing deny_rights property but it is talked about in the windows security section. https://docs.chef.io/resources/directory/#:~:text=Use%20the-,deny_rights,-property%20to%20deny

Reply:
Corrected